.eslintrc.js

```diff
@@ -2,11 +2,15 @@ module.exports = {
   extends: [
     'eslint:recommended',
     'plugin:react/recommended',
-    'plugin:prettier/recommended',
     'airbnb',
+    'plugin:prettier/recommended',
   ],
-  plugins: ['prettier', 'react'],
+  plugins: ['react', 'prettier'],
   rules: {
     'prettier/prettier': ['error'],
+    'react/jsx-filename-extension': [1, { extensions: ['.js'] }],
```

Comment:
Since all the component files use the `.js` extension, not `.jsx`, the rule needs to be modified.